test/viewModels/DepartureStatusSpec.scala

```diff
@@ -61,7 +68,6 @@ class DepartureStatusSpec extends SpecBase with Generators with ScalaCheckProper
     }
 
     "When status is guaranteeValidationFail show correct status and action" in {
```

Comment:
Consider having another test for TransitDeclarationRejected that checks the href (similar to the guaranteeValidationFail)

Reply:
This is covered in the first test!